### PR TITLE
FDP-590 ~ Adds fixes for failing cucumber tests in nightly build

### PR DESCRIPTION
**Pull request by smvdheijden**
Test all (not just @TestThisInPR)
Signed-off-by: Sander van der Heijden <sander.van.der.heijden@alliander.com>

### CODE REVIEW

**Comment by smvdheijden** [comment]
Nightly Build is running against feature branch in order to test the `@NightlyBuildOnly` scenarios. 
See https://ci.opensmartgridplatform.org/view/Nightly/job/OSGP_open-smart-grid-platform_nightly/1570/

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java [hunk 1]
```diff
@@ -9,31 +9,36 @@
...
   public WsCoreNotificationDatabase(
-      final CoreResponseDataRepository responseDataRepository,
-      final CoreResponseUrlDataRepository responseUrlDataRepository,
-      final CoreNotificationWebServiceConfigurationRepository
-          notificationWebServiceConfigurationRepository,
-      final CoreApplicationKeyConfigurationRepository applicationKeyConfigurationRepository,
-      @Value("${web.service.notification.port}") final int webServiceNotificationPort,
-      @Value("${web.service.notification.context}") final String webServiceNotificationContext) {
+      @Qualifier("wsCoreResponseDataRepository")
+          final ResponseDataRepository responseDataRepository,
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java:25]: Is het niet fraaier om hier niet de Qualifier te gebruiken maar de naam van het argument anders te maken.... `          final ResponseDataRepository wsCoreResponseDataRepository,`

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreResponseUrlDataRepository.java [hunk 2]
```diff
@@ -12,5 +12,5 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.ResponseUrlDataRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsCoreResponseUrlDataRepository")
 public interface CoreResponseUrlDataRepository extends ResponseUrlDataRepository {}
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreResponseUrlDataRepository.java:16]: Heb je overwogen om de naam van de class: WsCoreResponseUrlDataRepository te maken?

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreApplicationKeyConfigurationRepository.java [hunk 3]
```diff
@@ -12,6 +12,6 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.ApplicationKeyConfigurationRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsCoreApplicationKeyConfigurationRepository")
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreApplicationKeyConfigurationRepository.java:15]: Heb je overwogen om de naam van de class WsCoreApplicationKeyConfigurationRepository te maken (dit geldt voor meerdere classes in dit package)

File: integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/database/ws/DistributionAutomationNotificationWebServiceConfigurationRepository.java [hunk 4]
```diff
@@ -12,6 +12,6 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.NotificationWebServiceConfigurationRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsDistributionAutomationNotificationWebServiceConfigurationRepository")
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/database/ws/DistributionAutomationNotificationWebServiceConfigurationRepository.java:15]: Heb je overwogen om de naam van de class WsDistributionAutomationNotificationWebServiceConfigurationRepository te maken (dit geldt voor meerdere classes in dit package)

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java (hunk 1 above)
Comment (bvdzwet) [reply integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java:25]: Please stick to English when working or commenting on pull requests in the OSGP repositories.

Comment (kroesctrl) [reply integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java:25]: I changed the comments to English

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreResponseUrlDataRepository.java (hunk 2 above)
Comment (smvdheijden) [reply integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreResponseUrlDataRepository.java]: I have renamed this class and the other repository classes as well.

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreApplicationKeyConfigurationRepository.java (hunk 3 above)
Comment (smvdheijden) [reply integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreApplicationKeyConfigurationRepository.java]: I have renamed this class and the other repository classes as well.

File: integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java (hunk 1 above)
Comment (smvdheijden) [reply integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java]: Personally I'd like to keep the variable names somewhat shorter. I have renamed the repository classes to get rid of the qualifiers here.

**Pull request merged**